src/commands/alias/set.ts

```diff
@@ -21,12 +20,14 @@ export default class Set extends AliasCommand {
 
   public async run(): Promise<AliasResult[]> {
     const varargs = this.varargs || {};
-    const valuesToSet = Object.keys(varargs).map((v) => `${v}=${varargs[v] as string}`);
-    const savedValues = (await Aliases.parseAndUpdate(valuesToSet, AliasGroup.ORGS)) as Dictionary<string>;
-    const results = Object.keys(savedValues).map((alias) => ({
-      alias,
-      value: savedValues[alias],
-    }));
+    const info = await GlobalInfo.create();
```

Comment:
@WillieRuemmele why are you using `GlobalInfo.create()` vs `GlobalInfo.getInstance()`?

Reply:
probably just an artifact from `find + replace` - updating 👍